bessctl/conf/perftest/pktgen_http.bess

```diff
@@ -0,0 +1,55 @@
+import scapy.all as scapy
+from scapy.layers import http
+
+#Check shell variables for values, if not set to defaults.
+pkt_size = int($SN_PKT_SIZE!'1000')
+num_ports = int($SN_PORTS!'1')
+num_cores = int($SN_CORES!'1')
+
+assert(250 <= pkt_size <= 1522)
+assert(1 <= num_ports <= 16)
+assert(1 <= num_cores <= 4)
+
+# generate flows by varying IP dst 
+num_flows = int($SN_FLOWS!'1')
+assert(1 <= num_flows <= 10000)
+
+# Random ethernet address
+eth = scapy.Ether(src='02:1e:67:9f:4d:ae', dst='06:16:3e:1b:72:32')
```

Comment:
Could you make these configurable with env vars similar to SN_PORTS, SN_CORES, etc...? I think that will make using this in an automated way easier